recipes/taxbrain/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "taxbrain" %}
+{% set version = "2.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PSLmodels/Tax-Brain/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 34e0e98390ec7616df13e7c6d885a7e4213f3fa36295477bbb7155781800bdbc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - taxbrain = taxbrain.cli:cli_main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - taxcalc >=3.0.0
+    - behresp >=0.11.0
+    - dask
+    - bokeh
+    - markdown
+    - tabulate
+    - pypandoc
+    - matplotlib
```

`matplotlib-base` should be enough if you don't have a hard dependency on the QT backend.
```suggestion
    - matplotlib-base
```

---

Thanks for the suggestion, @xhochy! Changed in next commit